rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -292,3 +292,67 @@ def create_client(self):
         print "*********   call OvsVsctl.create_client"
         client = self._OvsVsctl(self.credential)
         return client
+
+
+
+@configure("ovs-ofctl")
+class OvsOfctl(OvsClient):
+
+    class _OvsOfctl(object):
+
+        def __init__(self, credential):
+            self.ssh = get_ssh_from_credential(credential)
+            self.context = {}
+            self.batch_mode = False
+            self.sandbox = None
+            self.cmds = None
+
+
+        def set_sandbox(self, sandbox, install_method="sandbox"):
+            self.sandbox = sandbox
+            self.install_method = install_method
+
+        def run(self, cmd, opts=[], args=[], stdout=sys.stdout, stderr=sys.stderr):
+            # TODO: add support for docker
+            self.cmds = self.cmds or []
```

Comment:
no batch mode for ovs-ofctl, so self.cmds is useless for this class. The run() function should simply self.ssh.run().